### PR TITLE
Add devdocs extension

### PR DESCRIPTION
**Pull request by pomdtr**
## Description

Search DevDocs from Raycast!

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/17577332/138616566-39e907a8-91a0-46c7-afed-e553f8ee40e3.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

**Comment by pomdtr** [comment]
This PR is ready for review @pernielsentikaer

File: extensions/devdocs/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    <List.Item
+      title={entry.name}
+      icon={icon}
+      key={entry.name + entry.path}
+      accessoryTitle={entry.type}
+      keywords={[entry.type].concat(entry.name.split("."))}
+      actions={
+        <ActionPanel>
+          <OpenInBrowserAction url={`${DEVDOCS_BASE_URL}/${doc.slug}/${entry.path}`} onOpen={() => popToRoot()} />
+          <OpenInDevdocsAction url={`${DEVDOCS_BASE_URL}/${doc.slug}/${entry.path}`} onOpen={() => popToRoot()} />
```
Comment (pernielsentikaer) [inline extensions/devdocs/src/index.tsx:141]: Seems like they do the same?

File: extensions/devdocs/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "license": "MIT",
+  "name": "devdocs",
+  "title": "Devdocs",
+  "description": "Search through DevDocs documentation sets from Raycast.",
+  "icon": "devdocs.png",
+  "author": "pomdtr",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Devdocs",
```
Comment (pernielsentikaer) [inline extensions/devdocs/package.json:12]:
```suggestion
      "title": "Search Documentation",
      "subtitle": "DevDocs"
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @pomdtr 👋 

Awesome extension, looking forward to use this when it's released 💯 

A few suggestions left before merging the gem 💎 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/devdocs/src/index.tsx (hunk 1 above)
Comment (pomdtr) [reply extensions/devdocs/src/index.tsx:141]: The second one open devdocs in the unofficial MacOS desktop app https://github.com/dteoh/devdocs-macos

Comment (pomdtr) [reply extensions/devdocs/src/index.tsx:142]: I added a redirection to the github page if the app is not currently installed.

Comment (pernielsentikaer) [reply extensions/devdocs/src/index.tsx:142]: Ah, didn't see that one, my bad! Awesome addition though!

**Comment by pernielsentikaer** [comment]
Awesome @pomdtr 💪 

That was the missing bit 💯

**Pull request merged**